docs/OpenTypeFeatureFileSpecification.md

```diff
@@ -3866,14 +3866,14 @@ along with the tag `sbit`.
 **v1.18 [16 Mar 2016]:**
 
 *   Clarified use of commas in ignore statement.
-*   Fixed spelling errors: example at end of section 9.e name table, example 1
-    in section 5.f.ii ignore sub.
+*   Fixed spelling errors: example at end of section [9.e](#9.e) name table, example 1
+    in section [5.f.ii](#5.f.ii) ignore sub.
 *   Increased maximum length of glyph names and named glyph classes from 31 to 63.
 
 **v1.17 [6 Jan 2016]:**
 
-*   Fixed bug in example for salt feature in section 8.a, and in example for
-    contextual lookup specification in section 6.h.ii.
+    contextual lookup specification in section [6.h.ii](#6.h.ii).
```

Comment:
Are these two lines (3875 and 3876) accidentally swapped?

Reply:
So weird. No idea how that happened.